gstd/src/lib.rs

```diff
@@ -31,6 +31,8 @@
 #[cfg(target_arch = "wasm32")]
 extern crate galloc;
 
+extern crate alloc;
```

Comment:
alloc and btreemap is already exported in prelude mod

Reply:
ok removed